<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.1 (Python 3.12.3 on win32)" generated="2024-10-22T10:36:07.671650" rpa="false" schemaversion="5">
<suite id="s1" name="Robot Interactive Console" source="c:\Users\josef\.vscode\extensions\robocorp.robotframework-lsp-1.13.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot">
<test id="s1-t1" name="Default Task/Test" line="5">
<kw name="Interpreter Main Loop" owner="MainLoop">
<status status="PASS" start="2024-10-22T10:36:07.710522" elapsed="2.056509"/>
</kw>
<status status="PASS" start="2024-10-22T10:36:07.709521" elapsed="2.057510"/>
</test>
<status status="PASS" start="2024-10-22T10:36:07.674226" elapsed="2.094227"/>
</suite>
<statistics>
<total>
<stat pass="1" fail="0" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="1" fail="0" skip="0" id="s1" name="Robot Interactive Console">Robot Interactive Console</stat>
</suite>
</statistics>
<errors>
<msg time="2024-10-22T10:36:07.661467" level="WARN">Error in file 'c:\Users\josef\.vscode\extensions\robocorp.robotframework-lsp-1.13.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot' on line 4: Singular section headers like '*** Test Case ***' are deprecated. Use plural format like '*** Test Cases ***' instead.</msg>
</errors>
</robot>
